apps/sertra/components/HeadWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import Head from "next/head";
+
+interface HeadProps {
+  title: string;
+  description: string;
+}
+
+export const HeadWrapper: React.FC<HeadProps> = ({ title, description }) => (
+  <Head>
+    <title>{title}</title>
```

Comment:
This could be ```{`${title} | Thunderstore`}``` so we don't need to repeat it everywhere.

Reply:
True, I'll add it there 👍